source/material_model/visco_plastic.cc

```diff
@@ -1072,38 +1072,31 @@ namespace aspect
                                    "$\\hat{D} = \\hat{D_{e}} + \\hat{D_{v}}$.  "
                                    "These terms further decompose into "
                                    "$\\hat{D_{v}} = \\frac{\\tau}{2\\eta}$ and "
-                                   "$\\hat{D_{e}} = \\frac{\\overset{\\triangledown}{\\tau}}{2\\mu}$, where "
+                                   "$\\hat{D_{e}} = \\frac{\\overset{\\nabla}{\\tau}}{2\\mu}$, where "
                                    "$\\tau$ is the viscous deviatoric stress, $\\eta$ is the shear viscosity, "
-                                   "$\\mu$ is the shear modulus and $\\overset{\\triangledown}{\\tau}$ is the "
-                                   "Jaumann corotational stress rate. If plasticity is included the deviatoric "
-                                   "rate of deformation may be written as: "
-                                   "$\\hat{D} = \\hat{D_{e}} + \\hat{D_{v}} + \\hat{D_{p}}$, where $\\hat{D_{p}}$ "
-                                   "is the plastic component. As defined in the second paragraph, $\\hat{D_{p}}$ "
-                                   "decomposes to $\\frac{\\tau_{y}}{2\\eta_{y}}$, where $\\tau_{y}$ is the yield "
-                                   "stress and $\\eta_{y}$ is the viscosity rescaled to the yield surface. "
-                                   "\n\n "
```

Comment:
What was the motivation for removing lines 1079-1084?

Reply:
I meant to move it down by one line, but I also need to remove the word "as defined in the second paragraph" because I have no idea what second paragraph this is referring to, and I don't think it's necessary. I'll make this fix and push the commit back.